iot/clients/src/main/java/io/enmasse/iot/transport/MessageDelivery.java

```diff
@@ -22,17 +22,17 @@
 public class MessageDelivery {
 
     private final String address;
-    private final String message;
+    private final byte[] data;
```

Comment:
We will bring only JSON string but I agree on having raw bytes as body message. Do you think that I should change the `send` method in the client as well ? Having `byte[ ]` and not just string as body to send ?

Reply:
Most encode/decode methods in json libraries use byte[] and not String (since json is UTF-8 encoded, java strings require some extra encode/decode within such libraries), so I changed it mostly for convenience. We can change send() also if you'd like